src/materialized/src/bin/materialized/main.rs

```diff
@@ -475,6 +486,25 @@ fn run(args: Args) -> Result<(), anyhow::Error> {
         let key = args.tls_key.unwrap();
         Some(materialized::TlsConfig { mode, cert, key })
     };
+    let mzcloud_params = if let Some(tenant_id) = args.mz_cloud_tenant {
+        let frontegg_jwk = match args.mz_cloud_frontegg_jwk {
+            None => bail!("expected mz_cloud_frontegg_jwk"),
+            // Attempt to base64 decode the jwk, otherwise keep it as is. This is
+            // unambiguous because the standard base64 alphabet does not contain dash,
+            // which is present in the key.
+            Some(jwk) => base64::decode(&jwk).unwrap_or_else(|_| jwk.as_bytes().to_vec()),
```

Comment:
Oh, interesting! Cloud currently passes the key as a literal PEM-encoded string on the environment (with newlines and all), no base64.

Reply:
(Oh, I just realized that this does an `unwrap_or_else`, reading the key verbatim if base64 fails. Looks good.